### PR TITLE
Bump flatpak to 0.9.11/12

### PR DESCRIPTION
**Pull request by ananace**
The 0.9.12 release fixes an issue that completely broke all Flatpak applications containing extradata

### CODE REVIEW

File: dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	>=sys-fs/libostree-2017.10
+	>=net-libs/libsoup-2.4
+	>=dev-libs/elfutils-0.8.12
+	>=dev-libs/glib-2.44:2
+	dev-libs/json-glib
+"
+DEPEND="${RDEPEND}
+	>=sys-devel/gettext-0.18.2
+	virtual/pkgconfig
+	>=dev-libs/gobject-introspection-1.40
```
Comment (fosero) [inline dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild]: gobject-introspection was [dropped](https://github.com/flatpak/flatpak-builder/commit/c011526e875d4429fbe266de7bc5b9040c05a5f2#diff-67e997bcfdac55191033d57a16d1408a)

**Review by fosero — CHANGES_REQUESTED** [review]
lgtm, one minor issue

File: dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild (hunk 1 above)
Comment (ananace) [reply dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild]: Ah, missed that, thanks.

Comment (fosero) [reply dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild]: And when updating the ebuild you also need to re-digest.

Comment (ananace) [reply dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild]:
```
$ sed -e '/0.9.11/d' -i dev-util/flatpak-builder/Manifest
$ ebuild dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild digest
Appending /home/ace/Projects/Gentoo/flatpak-overlay to PORTDIR_OVERLAY...
>>> Creating Manifest for /home/ace/Projects/Gentoo/flatpak-overlay/dev-util/flatpak-builder
$ git status
On branch bump-flatpak
Your branch is up-to-date with 'origin/bump-flatpak'.

nothing to commit, working tree clean
```

Comment (fosero) [reply dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild]: oh wait, the repo is on thin-manifest, so it isn't needed for the ebuilds.

One more thing however is that I usually just delete the old ebuild for the SLOT.

Comment (ananace) [reply dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild]: You seem to still have the 0.8.7 flatpak ebuild in place though?

I assume you mean for the minor version then?

Comment (fosero) [reply dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild]: yeah.. i shouldn't have said slot. I meant stable/unstable. I want to keep a stable version around for ppl who prefer it.

Comment (fosero) [reply dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild]: Could you squash the commits?

Comment (fosero) [reply dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild]: And separate them out per package? (1 commit for builder, 1 commit for general)

**Comment by ananace** [comment]
Oops, rebased onto one commit too early

**Comment by ananace** [comment]
Should be good to merge now I hope

**Review by fosero — APPROVED** [review]

**Comment by fosero** [comment]
Thanks for the patches.

**Pull request merged**